lib/src/mock_platform_adapter.dart

```diff
@@ -0,0 +1 @@
+
```

Comment:
intending to leave this in and complete the mock story?  or should this be deleted to handle the mock piece separately?